media/changes.xml

```diff
@@ -36,6 +36,9 @@
       <action type="remove" dev="sseifert">
         DamVideoMediaMarkupBuilder: Do not generate flash player as fallback.
       </action>
+      <action type="update" dev="mrozati" issue="WHAN-33">
```

Comment:
please rebase with current develop branch and create a new section for release 1.9.0 - 1.8.2 was already released.

Reply:
done